src/app/core/services/user/user.service.ts

```diff
@@ -34,6 +34,10 @@ export class UserService {
     return this.hasRole('SUPER_ADMIN');
```

Comment:
I think it would be better to rename this method to `isSuperAdmin()` to be more clear and avoid confusion